config/rabe-fs-backup.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run RaBe Backup Service timer daily
+
+[Timer]
+# Run once per day
+OnCalendar=daily
+# Start immediately if last start was missed (ie when machine was down)
+Persistent=true
+# Delay timer randomly to spread out load (centos-7.3 and up only)
+RandomizedDelaySec=30min
+# Do not coalesce with other timers, no need to save power during a backup
+AccuracySec=1us
+# try waking system if possible
+WakeSystem=true
+# Specify the service to be run
+Unit=airtime-backup.service
```

Comment:
Shouldn't the unit be named `rabe-fs-backup.service `? If yes, you can remove it, as it will [default](https://www.freedesktop.org/software/systemd/man/systemd.timer.html#Unit=) to this value anyway.